modules/rosa-hcp-classic-comparison.adoc

```diff
@@ -58,6 +58,8 @@
 * US East - Ohio (us-east-2)
 * US West - Oregon (us-west-2)
 * Africa - Cape Town (af-south-1)
+* Asia Pacific - Tokyo (ap-northeast-1)
```

Comment:
Consider sorting the list by region to make the list more scanable. All "northeast" are not grouped.

Reply:
That's intentional. I was matching AWS's presentation: https://docs.aws.amazon.com/general/latest/gr/rosa.html